backends-velox/src/main/scala/org/apache/gluten/backendsapi/velox/VeloxIteratorApi.scala

```diff
@@ -145,10 +146,10 @@ class VeloxIteratorApi extends IteratorApi with Logging {
 
   private def getPartitionColumns(
       schema: StructType,
-      partitionedFiles: Seq[PartitionedFile]): Seq[JMap[String, String]] = {
+      partitionedFiles: Seq[PartitionedFile]): Seq[mutable.Map[String, String]] = {
```

Comment:
If no further write operations on the Map, can we just return Seq[Map[String, String]]?